specify/specify7
# Properly set cellMeta of cells in WorkBench after validation

Fixes #3635 

The issue stemmed from the fact that `throttle` was now calling the `updateCellInfoStats` method, which re-bound its context and set it to undefined rather than WBView. 
Preferably, I would like to set the context of the function without passing it as a parameter (although this does allow for greater customization). 
I would love to hear other developers thoughts on this. 

To Test:
- Ensure the functionality of the Cell Navigation Buttons (Search Results, Modified Cells, New Cells, Error Cells) remain constant through the Workbench operations do not exhibit any unusual behavior
- Lightly ensure that the rendering behavior of the WorkBench when resizing your browser window remains the same as it did before Specify v7.8.12
